### PR TITLE
gRPC doc RHOL-1065

### PR DESCRIPTION
**Pull request by dckc**
## Overview

Add some doc to .proto files and extract in .md format.

### Which JIRA issue does this PR relate to?

https://rchain.atlassian.net/browse/RHOL-1065

### Complete this checklist before you submit the PR
- [x] This PR contains no more than 200 lines of code, excluding test code.
- [ ] This PR meets [RChain development coding standards](https://rchain.atlassian.net/wiki/spaces/DOC/pages/28082177/Coding+Standards).
- [ ] If this PR adds a new feature, this PR includes tests related to this feature.
- [x] You assigned one person to review this PR

### If you are not a member of the core development team, please confirm:
- [x] You signed the commit. Merging requires a signature. Please see the [developer wiki](https://rchain.atlassian.net/wiki/spaces/DOC/pages/498630673/How+to+sign+commits+to+rchain+rchain) for instructions.
- [x] Your GitHub account is also an account with [Travis CI](https://travis-ci.org). Unit tests will not run on your PR unless you have an account with Travis. Merge requires passed unit tests.

### Notes

Rebuilding the .md file uses a protoc extension written in go. It's straightforward to install.

### CODE REVIEW

File: docs/rnode-api/index.md [hunk 1]
```diff
@@ -0,0 +1,1924 @@
...
+| version | [int64](#int64) |  |  |
+| deployCount | [int32](#int32) |  |  |
+| tupleSpaceHash | [string](#string) |  | Same as postStateHash of BlockMessage |
+| tupleSpaceDump | [string](#string) |  |  |
+| timestamp | [int64](#int64) |  |  |
+| faultTolerance | [float](#float) |  |  |
+| mainParentHash | [string](#string) |  |  |
+| parentsHashList | [string](#string) | repeated |  |
+| sender | [string](#string) |  |  |
+| shardId | [string](#string) |  |  |
```
[the remark this answers is not in the document]
Comment (dckc) [reply docs/rnode-api/index.md]: I don't understand... updated what? shardId is a string in the latest CasperMessage.proto (bd4b67b)

[the remark this answers is not in the document]
Comment (KentShikama) [reply docs/rnode-api/index.md]: preStateHash, postStateHash . There might be more.

[the remark this answers is not in the document]
Comment (dckc) [reply docs/rnode-api/index.md]: yes, your Nov 20 changes are included, including RChainState. I still don't understand the point of your comment. Are you suggesting that something is missing? Extra? Wrong?

File: docs/rnode-api/index.md [hunk 2]
```diff
@@ -0,0 +1,1924 @@
...
+### RChainState
+
+
+
+| Field | Type | Label | Description |
+| ----- | ---- | ----- | ----------- |
+| preStateHash | [bytes](#bytes) |  | hash of the tuplespace contents before new deploys |
+| postStateHash | [bytes](#bytes) |  | hash of the tuplespace contents after new deploys |
+| bonds | [Bond](#coop.rchain.casper.protocol.Bond) | repeated | Internals of what will be the &#34;blessed&#34; PoS contract (which will be part of the tuplespace in the real implementation). |
+| blockNumber | [int64](#int64) |  |  |
```
Comment (KentShikama) [inline docs/rnode-api/index.md]: Ah ok here they are. I was looking at something different.

**Pull request merged**